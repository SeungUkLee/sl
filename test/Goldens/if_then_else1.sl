(* if then else (type error) *)

if 1 == 0 then 0 else true 
