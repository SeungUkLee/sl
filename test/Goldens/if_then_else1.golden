"[Type error] unification error: \nexpected: int\nrecevied: bool"